src/app/medicine/medicine.cpp

```diff
@@ -0,0 +1,26 @@
+#include "medicine.hpp"
+
+Medicine::Medicine(const std::string &name, const double price, const int amount, const int expirationDate,
+                   const std::pair<double, double> &storageTemperature, const std::vector<std::string> &composition)
+    : Product(name, price, amount), expirationDate_(expirationDate), storageTemperature_(storageTemperature),
+      composition_(composition)
+{
+}
```

Comment:
Please use initializer list in constructor

Reply:
Done